### ISSUE TITLE
conceal feature does not work properly

### ISSUE THREAD

**Issue by jdhao** [plasticboy/vim-markdown#395]
I opened a markdown file in Neovim and find that inline code, inline link and code blocks are rendered so that actual text is hidden. This is acceptable when I am in normal mode. But when I start editing the file, the actual source is still not shown, which is annoying (see image below).

![out](https://user-images.githubusercontent.com/16662357/47263420-f5e0d100-d533-11e8-82a3-35f308d6b9a4.jpg)

If I disable this plugin and open the markdown, I can the actual source text. How to fix this issue?

**Comment by thetarkus** [comment]
The `indentLine` plugin can break this functionality.
Otherwise `set concealcursor-=n` will not conceal the line that the cursor is on.

**Comment by jdhao** [comment]
It seems that using `set concealcursor-=n` inside `init.vim` does not work if I open a markdown with Neovim. I have to manually source `inti.vim` again to make the change take effect. It is a little strange.

**Comment by thetarkus** [comment]
Yeah, it doesn't work permanently for me in my .vimrc either. Strange indeed.

What fixed it for me was
`let g:indentLine_concealcursor = ''`

The only downside to this is that the identLine character won't appear on the line that your cursor is on, but it's easy to get over it as a workaround.

**Comment by rodhash** [comment]
> Yeah, it doesn't work permanently for me in my .vimrc either. Strange indeed.
> 
> A workaround is:
> `let g:indentLine_concealcursor = ''`
> 
> The only downside to this is that the identLine character won't appear on the line that your cursor is on, but it's easy to get over it as a workaround.
> 
> Another solution is to disable indentLine in markdown files:
> 
> ```viml
> augroup FILETYPES
>   autocmd FileType markdown let b:indentLine_enabled = 0
>   " ... other autocmds
> augroup END
> ```


I had a very annoying issue where the conceal chars (ie: `) were automatically hidden and only visually selecting the line would allow me to see it.

Thank you very much for this workaround, worked just fine!
